src/Server/AllServers.ts

```diff
@@ -200,10 +200,7 @@ export function loadAllServers(saveString: string): void {
 }
 
 function excludeReplacer(key: string, value: any): any {
-  if (key === "runningScripts") {
-    return [];
-  }
-  return value;
+  return key === "runningScripts" && Array.isArray(key) ? [] : value;
```

Comment:
Shouldn't you be checking value, not key?

Reply:
Yeah my bad. I see what happened here, I had it right in some staged changes on a separate branch but then I fat fingered it when I made a new branch for the change (since it wasn't really related to the other branch)
![image](https://user-images.githubusercontent.com/84951833/236057292-f94e928e-bc1e-4fc1-845b-7e78b30112d0.png)